templates/airflow2.cfg.j2

```diff
@@ -474,7 +474,7 @@ error_logfile = {{ airflow_webserver_error_logfile }}
 # Access log format for gunicorn webserver.
 # default format is %%(h)s %%(l)s %%(u)s %%(t)s "%%(r)s" %%(s)s %%(b)s "%%(f)s" "%%(a)s"
 # documentation - https://docs.gunicorn.org/en/stable/settings.html#access-log-format
-access_logformat = {{ airflow_webserver_access_logformat }}
+access_logformat = {% if airflow_webserver_access_logformat %}{{ airflow_webserver_access_logformat }}{% endif %}
```

Comment:
I think this conditional should cover the whole line, just as line 411

Reply:
In this case it was intended to be like that because at the default cfg file of airflow it comes in blank

Reply:
ok, closing conversation then :) thanks!